lib/components/molecules/Form/DragDropInput/DragDropInput.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Label from '../../../atoms/Label';
+import Input from '../../../atoms/Input';
+import styles from './DragDropInput.style';
+import useOpenFileDialog from '../../../../hooks/useOpenFileDialog';
+import useDragandDrop from '../../../../hooks/useDragandDrop';
+import type { DragDropInputProps } from './types';
+
+const { openFileDialog } = useOpenFileDialog();
+
+const DDFieldInput = ({
+  id,
+  className,
+  type,
+  name,
+  label,
+  onChange,
+  onBlur,
+  labelProps,
+  fieldProps,
+  localeText,
+  ...others
+}: DragDropInputProps): Node => {
+  const fileInput = React.createRef();
+  const {
+    onDragOver,
+    onDragLeave,
+    onDrop,
+    handleKeyPress,
+    onFilesAdded,
+    getHighlightClass,
+  } = useDragandDrop();
+  const styleCls = getHighlightClass(className);
+
+  return (
+    // TODO: Add support for label position
+    <div
+      className={styleCls}
+      onClick={() => openFileDialog(fileInput)}
+      onDragOver={onDragOver}
+      onDragLeave={onDragLeave}
+      onDrop={onDrop}
+      onKeyDown={e => handleKeyPress(e, fileInput)}
+      role="button"
+      tabIndex="0"
+    >
+      <Label htmlFor={id} {...labelProps}>
+        {label}
+      </Label>
+      <Field
+        id={id}
+        name={name}
+        type={type}
+        render={({ field, form }) => {
+          const { handleChange, handleBlur } = form;
+          return (
+            <Input
+              type={type}
```

Comment:
Is type required here as we will always use "file" as type?

Reply:
Changed to File type.